service/scheduler.go

```diff
@@ -158,6 +158,10 @@ func (s *Scheduler) activateWorkflow(
 	wf table.Workflow,
 ) string {
 	if err := client.Activate(swf.OrchardID); err != nil {
+		isActivated, err2 := client.IsActivated(swf.OrchardID)
```

Comment:
where is the infinite loop coming from? it's activated but sprinkler think it's not?